packages/data-context/src/actions/MigrationActions.ts

```diff
@@ -28,6 +31,7 @@ import {
 } from '../sources/migration'
 import { makeCoreData } from '../data'
 import { LegacyPluginsIpc } from '../data/LegacyPluginsIpc'
+import resolveFrom from 'resolve-from'
```

Comment:
any reason we are using this and not `require.resolve`? https://github.com/sindresorhus/resolve-from/issues/8

we use `require.resolve` with `cwd` elsewhere to find the user's `typescript`: https://github.com/cypress-io/cypress/blob/6e03e020682e73e0e5480fe010fe818fce488205/npm/webpack-preprocessor/lib/typescript-overrides.ts#L20-L27

Reply:
Hm, looks like `resolve-from` should be obsolete and we can remove it entirely from this code base. Good point.

Reply:
@flotwig `require.resolve` is a little different, it gets the actual path that will be require'd (eg `index.js`). In this case I want `package.json`, so `resolveFrom` is a little better, it'll find the folder, not a specific file.

Reply:
Right 🤦 

I made the change here and also in another part of the code base and removed the `resolve-from` dependency from the monorepo. https://github.com/cypress-io/cypress/pull/21255/commits/56e615452957f81b0d879c772577fa680c367fb5